content/intermediate/operations/understanding-changesets/index.md

```diff
@@ -39,9 +70,48 @@ Let’s get started!
     8. In the review page, choose **Create Stack**.
     9. Refresh the stack creation page until you see your stack in the `CREATE_COMPLETE` status.
 
+   ::::
+   :::::
+
 ### Lab part 1
 In this part of the lab, you will specify a property, for a given resource type, that requires [no interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt) on stack updates. You will then create a change set to preview the changes, and inspect the output of the change set operation.
 
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+	
+Open the `bucket.yaml` CloudFormation template in your `Cloud9` editor, and add `VersioningConfiguration` as shown below. Save the file.
+
+```yaml
+MyS3Bucket:
+  Type: AWS::S3::Bucket
+  Properties:
+    BucketName: !Ref BucketName
+    VersioningConfiguration:
+      Status: Enabled
+```
+Next, create your first change set:
+
+    1. From Terminal run the following command to **Create change set**, specify a name for the change set for example:`bucket-versioning-update` and provide the `BucketName` parameter that chosen in above example.
+    :::code{language=shell showLineNumbers=false showCopyAction=true}
+    
+    aws cloudformation create-change-set \
+        --stack-name changesets-workshop \
+        --change-set-name bucket-versioning-update
+        --template-body file://bucket.yaml \
+        --parameters ParameterKey=BucketName,ParameterValue='TYPE_UNIQUE_BUCKET_NAME-HERE'
+    :::
+
+    2. CloudFormation returns the following output of the AWS CLI.
+        ```json
+        {
+        "StackId" : "arn:aws:cloudformation:us-east-1:123456789012:stack/changesets-workshop/330b0120-1771-11e4-af37-50ba1b98bea6",
+        "Id": "arn:aws:cloudformation:us-east-1:123456789012:changeSet/bucket-versioning-update/a470cff7-cb2c-4cba-bf27-2b3b9ccc1333"
+        }
+        ```
+    ::::
+
+::::tab{id="local" label="Local development"}    
+
 Open the `bucket.yaml` CloudFormation template in your favorite text editor, and add `VersioningConfiguration` as shown below. Save the file.
 
 ```yaml
```

Comment:
I get no indentation when viewing this locally, but using a `code` block works
***Also, please update closing `:::` on line 124***
```suggestion
:::code{language=yaml showLineNumbers=true showCopyAction=true}
```